assets/src/edit-story/app/story/useStoryReducer/reducers/arrangeElement.js

```diff
@@ -62,20 +62,20 @@ function arrangeElement(state, { elementId, position }) {
 
   const page = state.pages[pageIndex];
 
-  // Abort if there's less than two elements (nothing to rearrange)
-  if (page.elements.length < 2) {
+  // Abort if there's less than three elements (nothing to rearrange as first is bg)
+  if (page.elements.length < 3) {
     return state;
   }
 
   const currentPosition = page.elements.findIndex(
     ({ id }) => id === idToArrange
   );
 
-  if (currentPosition === -1 || page.backgroundElementId === idToArrange) {
+  if (currentPosition === -1 || page.elements[0].id === idToArrange) {
```

Comment:
Would be cleaner if we hid this behind a `getBackgroundElement` util instead of accessing the `0`th element

Reply:
I'm still considering something to this effect. Maybe both have `page.elements[0]` and `page.backgroundElement` point to the same object? It's stupid for serialization purposes of course, where we should remove one of them, but otherwise might make sense.